### ISSUE TITLE
Streamline the process of getting these items printed

### ISSUE THREAD

**Issue by AllenEllis** [AllenEllis/EverySingleMonth#4]
From a Twitter DM:

> A way to easily get these images to be printed up as sticky notes through vistaprint or other providers

I've searched into Vistaprint along with a lot of other providers and can't figure out how to do this, but I think it's really important. I wish that they had an endpoint where I could prefill information and a URL to the generated image, but I haven't found any online printing vendor support this programatically.

If we can find one, then it would be great to have a "print now" button after the image is generated that would direct users to the third party print provider with dimensions & image data prefilled.

**Comment by axelson** [comment]
Walgreens might be a possibility

https://developer.walgreens.com/sites/default/files/v3_Native_PhotoPrintsAPI.html

**Comment by AllenEllis** [comment]
> Walgreens might be a possibility
> 
> https://developer.walgreens.com/sites/default/files/v3_Native_PhotoPrintsAPI.html

Thanks, do you see any way to leave users with the abilitiy to pay and set contact info themselves?

So far, every API I've found (including this from what I gleam skimming the docs) is that they expect me to provide customer data and payment info -- handling the entire checkout process.

I really wish I could just direct our users to their website, with a couple of print details prefilled (size & image) by a URL, and be completely hands-off from the checkout process.

**Comment by axelson** [comment]
Yeah that doesn't sound like quite like we want. But it does look like Walgreens has an "Add to Cart" API where the user would finish the checkout on Walgreens itself (which I believe would also mean that we don't need to handle PII): https://developer.walgreens.com/api/addtocart/rest

Actually it looks like "Photo Prints: HTML Checkout API" is exactly what we want:
> The Photo Prints: HTML Checkout API allows any create/edit/share/store photo application developers the ability to print photos to any of the 8,000+ Walgreens and Duane Reade stores for Same-Day pickup.

https://developer.walgreens.com/api/photoprints/rest

Haven't had a chance to fully dig into that API but it seems like we can upload a photo to Wallgreens and then have them printed for same-day pickup.
